website/docker/README.md

```diff
@@ -0,0 +1,68 @@
+# thingsboard.github.io
+
+Based on [ruby:2.7.7](https://hub.docker.com/_/ruby). For detailed info, see the origin repository.
```

Comment:
This is really strange introduction sentence... Please improve it and add more user friendly explanation, so users will be aware of what is going to be later in the README from the beginning.

Reply:
Introduction modified.